src/libs/feature-previews-config.ts

```diff
@@ -89,12 +88,6 @@ const featurePreviewsConfig: readonly FeaturePreview[] = [
       'Feedback on Azure Collaborative Workflows experience'
     )}`,
   },
-  {
-    id: ENABLE_AZURE_COLLABORATIVE_WORKFLOW_RUNNERS,
```

Comment:
TOL: Should the other flag `ENABLE_AZURE_COLLABORATIVE_WORKFLOW_READERS` be renamed to maybe `ENABLE_AZURE_COLLABORATIVE_WORKFLOW` to reflect that both "reader" and runners are supported?

Reply:
Good point, I will rename the variable but leave the underlying id the same so that users do not need to re-select the feature flag.

Reply:
This change actually would end up touching a lot of files... 🤔 Maybe we can leave it for now?

Reply:
Yeah I didn't realize it would touch a lot of files. Since this isn't user facing I don't mind leaving it as it is 👍